pages/learn/accounts/enterprise-sso.md

```diff
@@ -47,36 +47,35 @@ __Note__: If you unlink the default team in the Identity Provider configuration,
 
 To log in using your enterprise SAML authentication, you must first have or [create](https://dashboard.balena-cloud.com/signup) a standard balenaCloud account using your company email address. Once logged in to this account, navigate to your [user preferences](https://dashboard.balena-cloud.com/preferences/details) and click "Enable" in the "Enterprise SSO" section.
 <!-- TODO: Update to latest screenshot -->
-<img alt="User preferences with enable SSO button highlighted" src="/img/common/saml/merge-account-accept.png" width="100%">
+<img alt="User preferences with Enable SSO button highlighted" src="/img/common/saml/merge-account-accept.png" width="100%">
 
 Next, provide the company `SSO Identifier` supplied by your balenaCloud organization administrator.
 <!-- TODO: Update to latest screenshot -->
 <img alt="Enable SSO modal with SSO identifier filled in." src="/img/common/saml/add-sso-identifier-merge-modal.png" width="60%">
 
-__Important:__ By activating SAML, you are transferring your personal account to a company account, and this action is non-reversible. The following changes will occur:
+__Important:__ By activating SAML, you are transferring your personal account to a company account. **This action is non-reversible**. The following changes will occur:
 * **Your API keys will be deleted**
-* **This is a non-reversible action**
 * You will no longer be able to create new API keys
 * Ownership of your fleets and devices will be transferred to your company
 * Your company can revoke your access at any time
 * You will no longer be able to log in using `username` and `password`
 
-You will still be subject to balenaCloud's **terms of service** and **master service agreement**, with the following changes:
+You will still be subject to balenaCloud's **Terms of Service** and **Master Service Agreement**, with the following changes:
```

Comment:
Same — perhaps worth linking ToS and MAS pages here.

Relatedly, it’s worth considering removing these caveats from here (or if it’s important for the user to understand things to maybe rephrase it in prose), given that a user will not miss it when they try to enable SAML.

Reply:
Added links